tests/operator/operator.go

```diff
@@ -2573,7 +2527,14 @@ spec:
 
 		It("[test_id:6257]should accept valid cert rotation parameters", func() {
 			kv := copyOriginalKv()
-			patchKvCertConfig(kv.Name, certConfig)
+			certRotationStrategy := v1.KubeVirtCertificateRotateStrategy{}
+			certRotationStrategy.SelfSigned = certConfig
+			certConfigData, err := json.Marshal(certRotationStrategy)
+			Expect(err).ToNot(HaveOccurred())
+
+			data := []byte(fmt.Sprintf(`[{"op": "replace", "path": "/spec/certificateRotateStrategy", "value": %s}]`, string(certConfigData)))
```

Comment:
Would it make sense to implement these with the new `PatchSet` ?

Reply:
I don't want to overload the PR. it's not small already.

Reply:
I love small PRs (this one is already quite long for me to review). On the other hand, this commit makes these complex, error-prone jsons proliferate. But I think I'm fine with it.

Reply:
I totally agree about the risky json. But I think it's belong to a follow-up PR